testsuite/summarize.awk

```diff
@@ -138,14 +138,9 @@ function record_unexp() {
     record_unexp();
 }
 
-/^re-ran / {
-    if (in_test){
-        printf("error at line %d: found re-ran inside a test\n", NR);
-        errored = 1;
-    }else{
-        RERAN[substr($0, 8, length($0)-7)] += 1;
-        ++ reran;
-    }
+# Record anything else seen on a test result as a failure
+/=> .* / {
+    if (in_test) record_unexp();
 }
 
 END {
```

Comment:
```suggestion
END {
    if (in_test) record_unexp();
```

Reply:
Indeed, I made the change to your suggestion, thank you very much